docroot/sites/all/themes/ilr_theme/scss/layout/_ilr-at-70-homepage-layout.scss

```diff
@@ -52,7 +51,7 @@ body.subsite-front {
           @include grid-span(7,6);
         }
         ul {
-          left: 42%;
```

Comment:
I had been trying to use grid to position the ul but it was causing problems with mobile. I would prefer to use the grid.